src/components/gherkin/Attachment.tsx

```diff
@@ -106,19 +111,25 @@ function text(
   const body =
     attachment.contentEncoding === 'IDENTITY' ? attachment.body : base64Decode(attachment.body)
 
+  const attachmentTitle = attachment.fileName != null ? attachment.fileName : "Attached Text";
```

Comment:
Where there's no defined fileName, I think it would be useful to append the media type to the fallback message, e.g. `Attached Text (application/json)`.

Reply:
Also, I guess `??` would be perfectly used here rather than the ternary operator

Reply:
Agreed